Pipfile

```diff
@@ -8,6 +8,7 @@ django = "*"
 
 [dev-packages]
 flake8 = "*" 
+pytest-django = "*"
```

Comment:
what about pytest package?
how do you make it trigger for each PR? 
where is the test for example?
please look at my comment in:
https://github.com/redhat-beyond/commune/pull/31